test/integration/patients/patient/dashboard.js

```diff
@@ -853,7 +853,8 @@ context('patient dashboard page', function() {
       })
       .visit('/patient/dashboard/1')
       .wait('@routePatient')
-      .wait('@routePatientFlows');
+      .wait('@routePatientFlows')
+      .wait('@routeAllProgramActions');
```

Comment:
not waiting for `routeAll` sometimes had the tests ending prior to the 2nd check this is testing in the `onStart`